### PR TITLE
CommandBar: Passing item through onclick as expected

### LINKED ISSUE #5854: CommandBar does not pass item on onClick handler anymore
### Bug Report 
 
- __Package version(s)__: 6.42
- __Browser and OS versions__: (fill this out if relevant)
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: Blocking
 
Products/sites affected: Flow
 
#### Describe the issue:
 
The onClick handler used to pass in "item" for the button in CommandBar
 
#### Actual behavior:
 
The onClick handler passes "undefined" for the button in CommandBar
 
#### Expected behavior:
 
The onClick handler used to pass in "item" for the button in CommandBar and relevant data.
 
### If applicable, please provide a codepen repro:
 
https://codepen.io/anon/pen/djQNEW
If you click on "Menu1", you'll see that item is undefined. 

https://codepen.io/anon/pen/LBXWRj
Change the Fabric version to 5.63, and it supplies the button item with relevant data passed through.

### PR DESCRIPTION
**Pull request by micahgodbolt**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #5854
- [x] Include a change request file using `$ npm run change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: apps/vr-tests/src/common/Themes.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+export const DarkTheme = {
+  themePrimary: '#0078d4',
```
Comment (micahgodbolt) [inline apps/vr-tests/src/common/Themes.ts]: doh....left this in there somehow

File: packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx [hunk 2]
```diff
@@ -182,6 +182,17 @@ export class CommandBarBase extends BaseComponent<ICommandBarProps, {}> implemen
     return <CommandButtonType {...commandButtonProps as IButtonProps} />;
   };
 
+  private _onButtonClick(item: ICommandBarItemProps): (ev: React.MouseEvent<HTMLButtonElement>) => void {
+    return ev => {
+      if (item.inactive) {
+        return;
+      }
```
Comment (JasonGore) [inline packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx]: Is this also consistent with the old code? I think it'd be good to add a unit test for this too...

Comment (micahgodbolt) [reply packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx]: Let me double check. I pulled that check in as it was used in the past.

File: packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.test.tsx [hunk 3]
```diff
@@ -101,6 +101,30 @@ describe('CommandBar', () => {
...
+    const commandBar = mount(
+      <CommandBar
+        items={[
+          {
+            text: 'TestText 1',
+            key: 'TestKey1',
+            className: 'MenuItem',
+            onClick: (ev, item) => {
+              testValue = ev && item && item.text;
+            }
```
Comment (eveleens) [inline packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.test.tsx]: could add a test for passing data as well? (If the item has the data property, it'll pass in the data with the item)

File: packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx (hunk 2 above)
Comment (micahgodbolt) [reply packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx]: inactive is deprecated. it'll be removed next major release anyway.

**Pull request merged**